src/Constants.h

```diff
@@ -9,6 +9,12 @@
 
 #include <frozen/unordered_map.h>
 
+namespace control {
+enum class DriveMode;
+}
+
+using control::DriveMode;
```

Comment:
Looks unused. Also btw, `using` declarations in header files should usually be avoided, since it causes namespace pollution.